Bridge/DoctrineStatsBundle/DataCollector/DoctrineStatsCollector.php

```diff
@@ -545,7 +545,7 @@ protected function parseManagedEntities()
     {
         $return = [];
         /** @var EntityManagerInterface $entityManager */
-        foreach ($this->doctrine->getEntityManagers() as $entityManager) {
+        foreach ($this->doctrine->getManagers() as $entityManager) {
```

Comment:
`getManagers()` return instances of `ObjectManager`, but i need an instanceof `EntityManagerInterface` to call `getUnitOfWork()`. So i can't call `getManagers()` here